src/Api/Library/Shared/Communicate/Communicate.php

```diff
@@ -294,8 +294,7 @@ public static function sendJoinRequestAccepted($user, $projectModel, $website, D
 
     private static function sendTemplateEmail($to, $subject, $templateName, $vars, $website, DeliveryInterface $delivery = null)
     {
-        $senderEmail = 'no-reply@' . $website->domain;
-        $from = [$senderEmail => $website->name];
+        $from = 'no-reply@languageforge.org';
```

Comment:
I'm thinking this will fix the unit test failures.
```suggestion
        $from = ['no-reply@languageforge.org' => 'Language Forge'];
```